src/OpenTracing/NoopTracer.php

```diff
@@ -13,16 +12,31 @@ public static function create()
         return new self();
     }
 
-    public function startSpan(
+    /**
+     * {@inheritdoc}
+     */
+    public function startActiveSpan(
         $operationName,
         SpanReference $parentReference = null,
```

Comment:
Active Span needs no reference anymore, and this being the second argument will lead to users having `startActiveSpan('op1', null, ...);`

Why is there no array of options anymore? Ruby and Python have a list of arguments as well, but both languages support named arguments, so they could do `startActiveSpan('op1', tags=['foo' => 'bar']);` which PHP cannot do.